src/stateToHTML.js

```diff
@@ -418,6 +417,6 @@ function encodeAttr(text: string): string {
     .split('"').join('&quot;');
 }
 
-export default function stateToHTML(content: ContentState, options: ?Options): string {
-  return new MarkupGenerator(content, options).generate();
+export default function stateToHTML(content: ContentState, options: ?Options): string {
+  return new MarkupGenerator(content, options).generate();
```

Comment:
No change really, must be some spacing or something changed just be opening and saving the file in my editor. I couldn't manage to unstage the hunk either using SourceTree.